content/departments/product-engineering/engineering/cloud/devops/managed/v1.1/mi1-1_failover_process.md

```diff
@@ -0,0 +1,59 @@
+# Failover a managed instance v1.1 to another zone
+
+<span class="badge badge-note">SOC2/CI-110</span>
+
+In MI v1.1, all resources are still zonal, and managed instances are not automatically resilient to zone-wide failures. It's possible to failover resources to another zone in the event of GCP zonal failure, but it's a tedious and manual process. Perform the failover in the following order.
+
+## Failover Cloud SQL
+
+Use cases:
+
+- Cloud SQL current zone is down
+
+### Edit Cloud SQL location
+
+Open [sourcegraph/deploy-sourcegraph-managed] and open `$CUSTOMER/terraform.tfvars`. Add the following override:
+
+> Replace the zone with whichever that is available in the same region at the moment.
+
+```hcl
+cloud_sql_zone = "us-central1-a"
+```
+
+Apply the changes
+
+```sh
+terraform apply
+```
+
+It will take some time for Cloud SQL to be moved to a different zone.
+
+Commit your change and open a Pull Request.
+
+## Failover the compute instance (VM)
+
+<!-- TODO
+Validate this actualy works
+-->
+
+Configure env var
+
+```sh
+eval (mg --customer $CUSTOMER workon)
+export PROJECT_ID=$PROJECT_PREFIX-$CUSTOMER
+export INSTANCE_NAME=default-$OLD_DEPLOYMENT-instance
+```
+
+Locate the most recent snaphost of the **current** data disk, note the name of the snaphost as `SNAPSHOT_NAME`
```

Comment:
Typo fix
```suggestion
Locate the most recent snapshot of the **current** data disk, note the name of the snapshot as `SNAPSHOT_NAME`
```

The emphasis on current implies it is possible to have a more recent snapshot of a  disk not in use. I'm kind of confused by this. Could you give a brief explanation?